docs/source/guide/index.rst

```diff
@@ -30,14 +30,22 @@ General Feature Guides
    events
 
 
-Service Feature Guides
-======================
+Service Feature Guide
+=====================
 
 .. toctree::
-   :maxdepth: 3
+   :maxdepth: 1
 
    cloud9
-   dynamodb
-   s3
```

Comment:
I think for `dynamodb` and `s3` it would still be worth leaving these under the service feature guides mainly because these are guides users can follow from beginning to end to learn how to use the service and use any higher-level customizations for that service.

Reply:
This commit is only the first iteration in a longer term project to reorganize the multiple User Guides and sections. The current "Service Level Guide" will eventually include higher-level services, such as Cloud9, CDK, SAM, etc. (and be renamed accordingly). When the coverage of DynamoDB, S3, and other lower-level services expands sufficiently to justify being considered a "DynamoDB/S3 Dev Guide," the doc structure can be updated to accommodate it. Currently, no service includes enough explanatory text or code samples to justify being considered a "Guide." As such, it makes sense to combine all the S3 code examples in one section and move the DynamoDB examples to the Code Examples chapter as this commit does.

Reply:
Gotcha. Could we rename the "Service Feature Guide" section title to reflect the direction it is going? Given the documentation will be updated once this change gets merged and released, the current title may be a little confusing as they may be expecting guides for AWS services as opposed to guides to higher-level services/tooling.

Reply:
Okay, renamed it to Tools Guide for now.